capabilityquerier/pkg/capabilityquerier/capabilityquerier.go

```diff
@@ -113,12 +129,21 @@ func requestCapabilityStatement(ctx context.Context, fhirURL *url.URL, client *h
 		message.MIMETypes = append(message.MIMETypes, fhir3PlusJSONMIMEType)
 	}
 	if capResp != nil {
-		err = json.Unmarshal(capResp, &(message.CapabilityStatement))
+		err = json.Unmarshal(capResp, &(capabilityStatement))
 		if err != nil {
 			return err
 		}
 	}
 
+	switch endptType {
+	case "metadata":
+		message.HTTPResponse = httpResponseCode
+		message.CapabilityStatement = capabilityStatement
+	case "well-known":
```

Comment:
not a fan of "magic strings". can you make this an enum or similar at the top of the file?